os-eap-migration/added/launch/openshift-migrate-common.sh

```diff
@@ -29,8 +30,15 @@ function runMigration() {
   local success=false
   local message="Finished, migration pod has been terminated"
   ${JBOSS_HOME}/bin/readinessProbe.sh
+  local probeStatus=$?
 
-  if [ $? -eq 0 ] ; then
+  if [ $probeStatus -eq 0 ] && [ "$(type -t probePodLog)" = 'function' ]; then
+    # -- checking if server.log is clean from errors (only if function of the particular name exists)
+    probePodLog # calling function from partitionPV.sh
+    probeStatus=$?
```

Comment:
Have you been able to verify that this is useful?  By default, logging is sent to stdout.  This may also levy requirements on the pod for storage related to logging (e.g. emptyDir volume mount).

Reply:
yes, at my knowledge this is useful and currently I haven't found better way to find what is outcome of the unavailable resource. Even though I've created a jira with asking QA guys to create test for it: https://issues.jboss.org/browse/CLOUD-2519

On the asking about the server log probing it's not done by checking the file itself but by querying OpenShift API. That way those are data taken from `stdout` not from a file in particular.